src/scripts/modules/media/body/content-baked.less

```diff
@@ -240,7 +240,7 @@ h1.example-title .text {
     tr {
       td {
         vertical-align: middle;
-        padding: 1em 0em;
+        padding: 1em 0 1em 8px;
```

Comment:
should we use only one unit? Maybe just `em`?

Reply:
So `<th>` is set with `padding: @table-cell-padding;` which stands for `padding: 8px;`
Propably the best way will be to set `td` with the same way?

Reply:
Is it possible to change `padding: 1em 0 1em 8px;` to `padding: @table-cell-padding 0 @table-cell-padding;`? Or maybe we simply use ` padding: 1em 0 1em 1em;`?